### ISSUE TITLE
Issue pin not there when disabled

### ISSUE THREAD

**Issue by jozefdobos** [3drepo/3drepo.io#492]
When I was commenting on existing issue, I was wondering why no pin was there. I thought that I had no permissions to drop pins only to be told that pin can be dropped only once for an issue and it cannot be changed. Maybe what would be better would be to still keep the icon just make it grayed out hinting that the pin has already been "utilised" or something.

**Comment by paloknapo** [comment]
When a new issue is created, enable dropping pin and take screenshot even without title typed in. Save is allow only with title as it works now.
